app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        vaccine_info = visitor["vaccine"]
+        if vaccine_info["expiration_date"] < datetime.today().date():
```

Comment:
Consider using `datetime.date.today()` instead of `datetime.today().date()` for comparing the expiration date, as per the task description. This ensures consistency with the examples provided in the task.